files/xsuaa.yaml.hbs

```diff
@@ -0,0 +1,21 @@
+serviceInstances:
+    xsuaa:
+        name: {{appName}}-uaa
+        serviceOfferingName: xsuaa
+        servicePlanName: broker
+        parameters:
+            xsappname: {{appName}}
+            tenant-mode: shared
+            oauth2-configuration:
+                credential-types:
+                    - instance-secret
```

Comment:
@Pavan-SAP Should we default instance-secret?

Reply:
No, this does not make sense. Let's only keep binding-secret and perhaps consider adding x509.
Consumers can also override this right?

Reply:
Yeah, they can always override it in the values file.